Content.Server/Atmos/Piping/Other/EntitySystems/GasMinerSystem.cs

```diff
@@ -25,10 +29,14 @@ private void OnMinerUpdated(Entity<GasMinerComponent> ent, ref AtmosDeviceUpdate
         {
             var miner = ent.Comp;
 
+            if(!GetValidEnvironment(ent, out var environment))
```

Comment:
Whitespace

Reply:
If `GetValidEnvironment` returns false, this does not correctly set `Idle`, does it?